src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -119,7 +119,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       RestoreRecursive="$(RestoreRecursive)"
       RestoreForce="$(RestoreForce)"
       HideWarningsAndErrors="$(HideWarningsAndErrors)"
-      Interactive="$(NuGetInteractive)"/>
+      Interactive="$(NuGetInteractive)"
+      ReevaluateRestoreGraph="$(ReevaluateRestoreGraph)"/>
```

Comment:
spec: This is not documented anywhere in the specs. 

This is the reevaluate lock file, hose everything (well almost) right?

Reply:
this is the implementation part of `--reevaluate` flag passed with `dotnet.exe restore` Since this is not explicitly allowed or needed for msbuild hence not mentioned in the spec. 

And yes, it just skip existing lock file and re-evaluate all the dependencies.